src/fsharp/LanguageFeatures.fs

```diff
@@ -22,8 +22,7 @@ type LanguageFeature =
     | LanguageVersion46 = 0
     | LanguageVersion47 = 1
     | SingleUnderscorePattern = 2
-    | Nullness = 1000
```

Comment:
Don't remove old features.

Reply:
Nullness, and ScriptingPackageManagement  aren't real features, yet.